plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue

```diff
@@ -146,6 +146,19 @@
               :inline-help="translate('UsersManager_FirstSiteInlineHelp')"
             />
           </div>
+          <div id="plugin-extra" v-for="(refComponent, index) in componentExtensions" :key="index">
```

Comment:
Seems like the ID is not used, so I'd remove it or if really needed for anything, probably add a class.
```suggestion
          <div v-for="(refComponent, index) in componentExtensions" :key="index">
```

Reply:
The ID is used in tests for detection as to if the extensions are implemented in the Matomo version under test..  I'll switch to a class, I was 50/50 on this.

Reply:
Actually we should never use an `id` attribute in a `v-for` directive, unless it receives it's value based on the currently iterated element. If you have more than one element in `componentExtensions` you would end up having multiple elements with the same id.